### PR TITLE
Aggregate result data

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**

### CODE REVIEW

File: docker.md [hunk 1]
```diff
@@ -35,30 +38,19 @@ We are not maintaining a central copy for now. Till then please main your own co
...
-mysql -p
-```
-When prompted for a password enter the value in `MYSQL_ROOT_PASSWORD`  
-`Create database <value in DATABASE_NAME>`
-
-5. Restart your containers. This additional step is neccessary since there is no dependency between the containers. And django would try to connect to DB not there yet and fail.  
-`docker-compose restart emba`
-
-6. Exec into your emba-django container  
-`docker exec -it amos-ss2021-emba-service_emba_1 bash`
```
Comment (MaximilianWagner) [inline docker.md:59]: Why is this deleted ?

File: embark/uploader/views.py [hunk 2]
```diff
@@ -137,6 +125,9 @@ def start_analysis(request, refreshed):
             logger.error(form.errors)
             return HttpResponse("Invalid Form")
 
+    FirmwareForm.base_fields['firmware'] = forms.ModelChoiceField(queryset=FirmwareFile.objects)
+    DeleteFirmwareForm.base_fields['firmware'] = forms.ModelChoiceField(queryset=FirmwareFile.objects)
```
Comment (MaximilianWagner) [inline embark/uploader/views.py:129]: This is not needed anymore. Its in forms.py

File: .gitignore [hunk 3]
```diff
@@ -9,6 +9,7 @@ embark/dir-combined.yara
 embark/emba_logs/
 embark/uploader/migrations/
 embark/users/migrations/
+embark/logs/web.log
```
Comment (MaximilianWagner) [inline .gitignore:12]: this might causes the pipeline to fail.

**Review by MaximilianWagner — CHANGES_REQUESTED** [review]
Please fix the pipeline.

File: .gitignore (hunk 3 above)
Comment (diegiesskanne) [reply .gitignore]: fixed

File: docker.md (hunk 1 above)
Comment (ashutoshsingh0223) [reply docker.md:59]: Not needed anymore. On first build database will come up automatically.  New Variable `MY_SQL_DATABASE` added in env does this.

File: embark/uploader/views.py (hunk 2 above)
Comment (ashutoshsingh0223) [reply embark/uploader/views.py:129]: Do you want me to delete this?

File: docker.md (hunk 1 above)
Comment (MaximilianWagner) [reply docker.md:59]: Ok nice, didnt know.

File: embark/uploader/views.py (hunk 2 above)
Comment (MaximilianWagner) [reply embark/uploader/views.py]: I deleted them myself, just for clarification.

**Review by MaximilianWagner — APPROVED** [review]

**Comment by MaximilianWagner** [comment]
Can we SQUASH and merge now ?
